go/test/endtoend/tabletmanager/tablet_health_test.go

```diff
@@ -266,7 +266,7 @@ func TestHealthCheckDrainedStateDoesNotShutdownQueryService(t *testing.T) {
 	assert.Equal(t, "SERVING", rdonlyTablet.VttabletProcess.GetTabletStatus())
 
 	// Change from rdonly to drained and stop replication. (These
-	// actions are similar to the SplitClone vtworker command
+	// actions are similar to the old SplitClone vtworker command
 	// implementation.)  The tablet will stay healthy, and the
```

Comment:
```suggestion
	// implementation). The tablet will stay healthy, and the
```

(i'm **so** sorry, i cannot help myself)